### DISCUSSION TITLE
Establishment of "Love is in the air-Banksy NFT" DAO

### DISCUSSION THREAD

**Discussion by zybk70** [avalanche-foundation/ACPs#11] [Brainstorming]
One of the most expensive pieces of art in the world is currently on the Avalanche network, friends. In order to understand the importance of the subject, I wrote something about this subject on reddit. Please read this article first.

https://www.reddit.com/r/Avax/comments/17a82ej/avalanches_forgotten_treasure_love_is_in_the/

Let's discuss together what needs to be done. To summarize, my thoughts on this subject are as follows.

![image](https://github.com/avalanche-foundation/ACPs/assets/54598911/5ec4e2ef-be0c-4e64-b519-c94706c15c33)

This is a poster from their latest event, they are traveling all over Europe. But no one mentions Avalanche. According to the poster, the valuation was 10 million pounds, the work was divided into 10k pieces and soldon the Avalanche network for a fixed price of $1500. 1000 pounds is about apr 118 AVAX right now. However, in floor places there are currently 18 AVAX in opensea. Avaissance needs to focus on this issue and encourage holders from the fund by establishing a DAO for holders. We must protect this rare piece.

**Comment by iJaack** [comment]
Do we know **exactly** where it is stored onchain? What is the collection name?

**Reply by zybk70** [reply]
Avalanche https://snowtrace.io/token/0xf675a87397a6239eaf95ad948670a5b19d076c59
